### PR TITLE
Method in example does not exist

### PR DESCRIPTION
**Pull request by foxyblue**
It looks like the [method](https://github.com/airbnb/omniduct/compare/master...foxyblue:fix/quick-start-example?expand=1#diff-9d8a6f5b32d4a70a56734fd0eb50dd5bL149) in the docs was [removed and replaced](https://github.com/airbnb/omniduct/commit/0f9f823919f53180fe064922dc4b65f6e9bae3f6#diff-dff26fb1fc0fb47bf9c77cf98bcf73b6L80) with `register_from_config`.

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -159,8 +159,8 @@ Task 3: Persist service configuration for use in multiple sessions
     ...         remote: my_server
     ... """)
 
-    >>> %%presto_local
```
Comment (matthewwardrop) [inline docs/quickstart.rst]: This was actually correct. By default, duct magic functions are registered when you use `register_from_config`. Can you drop this patch?

**Comment by matthewwardrop** [comment]
Hi @foxyblue ! Thanks for the documentation fix. Was there some reason I'm not aware of why you removed the magic `%%presto_local`?

**Comment by foxyblue** [comment]
@matthewwardrop thanks for the review ^_^

The only reason I removed it was to be safe that I was providing a usage example I knew would work as expected. I haven't yet utilised the registered magics. But now I'm aware. Thanks again!

**Comment by matthewwardrop** [comment]
Thanks again!

**Pull request merged**